package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "5e-database",
+  "version": "1.0.0",
+  "description": "Database for 5e-srd-api",
+  "scripts": {
+    "lint": "eslint . --ext .js,.json",
+    "nuke-and-replace-db": "./scripts/nuke-and-replace-db.sh"
```

Comment:
could be something like:

`db:refresh`

as tasks grow its handy to namespace/keep them easy to remember.